zerver/openapi/zulip.yaml

```diff
@@ -19574,39 +19586,45 @@ components:
       additionalProperties: false
       description: |
         Dictionary containing details of a single onboarding step.
-
-        **Changes**: Before Zulip 8.0 (feature level 233), this was
-        named as `Hotspot`. One-time notice wasn't available earlier.
       properties:
         type:
           type: string
           description: |
             The type of the onboarding step. Valid values are either
-            'hotspot' or 'one_time_notice'.
+            `"hotspot"` or `"one_time_notice"`.
 
             **Changes**: New in Zulip 8.0 (feature level 233).
         name:
           type: string
           description: |
-            The name of the hotspot.
+            The name of the onboarding step.
         title:
           type: string
           description: |
-            The title of the hotspot, as will be displayed to the user.
+            The title of the onboarding step, as displayed to the user.
+
+            Only present for onboarding steps with type `"hotspot"`.
         description:
           type: string
           description: |
-            The description of the hotspot, as will be displayed to the
+            The description of the onboarding step, as displayed to the
             user.
+
+            Only present for onboarding steps with type `"hotspot"`.
         delay:
           type: number
           description: |
-            The delay after which the user should be shown the hotspot.
+            The delay, in seconds, after which the user should be shown
+            the onboarding step.
+
+            Only present for onboarding steps with type `"hotspot"`.
         has_trigger:
           type: boolean
           description: |
-            The has_trigger is used to identify if the hotspot will activate
-            only when some specific event occurs.
+            Identifies if the onboarding step will activate only when some
+            specific event occurs.
```

Comment:
This seems like it didn't end up getting used in the implementation of one-time notices. I wasn't sure if there was a plan to eventually use it for the hotspots tutorial?

Reply:
More likely to be for one-time-notices, but we can let the dust settle on that after we've deleted the hotspots, which is likely to happen in the next couple weeks. @prakhar1144 FYI.